Extension/src/LanguageServer/extension.ts

```diff
@@ -1403,7 +1403,9 @@ export async function preReleaseCheck(): Promise<void> {
 }
 
 export async function getIncludes(maxDepth: number): Promise<any> {
-    const includes = await clients.ActiveClient.getIncludes(maxDepth);
+    const tokenSource = new vscode.CancellationTokenSource();
+    const includes = await clients.ActiveClient.getIncludes(maxDepth, tokenSource.token);
```

Comment:
We'd 'fixed' LSP cancelation such that it triggers an exception on cancellation.  However, it doesn't trigger the exception that VS Code generally expects, which would be a `CancellationError`.  I'm not sure what the caller would expect in this case.  Do you know?

Reply:
Thanks, I've checked and updated this to use the same VS Code semantics as the chat context request.